frontend/app/src/components/settings/controls/SettingsOption.vue

```diff
@@ -85,7 +88,7 @@ const updateImmediate = async (newValue: any) => {
   emit('finished');
 };
 
-const update = useDebounceFn(updateImmediate, 1500);
+const update = useDebounceFn(updateImmediate, debounce);
```

Comment:
Doesn't that mean that now we have two ways to do non debounced messages?
updateImmediate is also exposed to do the same thing

Reply:
oh my bad, I didn't see that the `updateImmediate` was also exposed